models/user.go

```diff
@@ -12,6 +13,22 @@ type User struct {
 	HashedPassword string
 }
 
+func (u *User) Keywords() orm.QuerySeter {
+	orm := orm.NewOrm()
+	return orm.QueryTable(Keyword{}).Filter("user_id", u.Id)
+}
+
+func (u *User) PaginatedKeywords(keywords orm.QuerySeter, offset int, paginatesPer int) ([]*Keyword, error) {
+	userKeywords := []*Keyword{}
+
+	_, err := keywords.Limit(paginatesPer, offset).OrderBy("-id").All(&userKeywords)
+	if err != nil {
+		logs.Error("Keywords pagination failed: ", err)
+	}
+
+	return userKeywords, nil
+}
+
```

Comment:
IMO, This 2 functions are the function of the keyword model. User is a filter in this case.
What do you think?

Reply:
I actually wanted to binding with the user as these are keyword is user specific and we can call these function on `currentUser`directly like `currentUser.Keywords(), currentUser.PaginatedKeywords(...)`.

This is similar to Rails where we do like

```Ruby
# In user model user.rb
has_many: posts

# then we can call `posts` method on the user object
user.posts
```
What is your though?

Reply:
**Update:** I moved these function to Keyword model.